crates/sui-core/src/validator_tx_finalizer.rs

```diff
@@ -273,46 +274,55 @@ where
-    use std::sync::atomic::AtomicBool;
-    use std::sync::atomic::Ordering::Relaxed;
-    use std::sync::Arc;
     use sui_macros::sim_test;
     use sui_swarm_config::network_config_builder::ConfigBuilder;
     use sui_test_transaction_builder::TestTransactionBuilder;
-    use sui_types::base_types::{AuthorityName, ObjectID, SuiAddress, TransactionDigest};
-    use sui_types::committee::{CommitteeTrait, StakeUnit};
-    use sui_types::crypto::{get_account_key_pair, AccountKeyPair};
-    use sui_types::effects::{TransactionEffectsAPI, TransactionEvents};
-    use sui_types::error::SuiError;
-    use sui_types::executable_transaction::VerifiedExecutableTransaction;
-    use sui_types::messages_checkpoint::{
-        CheckpointRequest, CheckpointRequestV2, CheckpointResponse, CheckpointResponseV2,
+    use sui_types::{
+        base_types::{AuthorityName, ObjectID, SuiAddress, TransactionDigest},
+        committee::{CommitteeTrait, StakeUnit},
+        crypto::{get_account_key_pair, AccountKeyPair},
+        effects::{TransactionEffectsAPI, TransactionEvents},
+        error::SuiError,
+        executable_transaction::VerifiedExecutableTransaction,
+        messages_checkpoint::{
+            CheckpointRequest, CheckpointRequestV2, CheckpointResponse, CheckpointResponseV2,
+        },
+        messages_grpc::{
+            HandleCertificateRequestV3, HandleCertificateResponseV2, HandleCertificateResponseV3,
+            HandleSoftBundleCertificatesRequestV3, HandleSoftBundleCertificatesResponseV3,
+            HandleTransactionResponse, ObjectInfoRequest, ObjectInfoResponse,
+            SubmitTransactionRequest, SubmitTransactionResponse, SystemStateRequest,
+            TransactionInfoRequest, TransactionInfoResponse,
+        },
+        object::Object,
+        sui_system_state::SuiSystemState,
+        transaction::{
+            CertifiedTransaction, SignedTransaction, Transaction, VerifiedCertificate,
+            VerifiedSignedTransaction, VerifiedTransaction,
+        },
+        utils::to_sender_signed_transaction,
     };
-    use sui_types::messages_grpc::{
-        HandleCertificateRequestV3, HandleCertificateResponseV2, HandleCertificateResponseV3,
-        HandleSoftBundleCertificatesRequestV3, HandleSoftBundleCertificatesResponseV3,
-        HandleTransactionResponse, ObjectInfoRequest, ObjectInfoResponse, SystemStateRequest,
-        TransactionInfoRequest, TransactionInfoResponse,
-    };
-    use sui_types::object::Object;
-    use sui_types::sui_system_state::SuiSystemState;
-    use sui_types::transaction::{
-        CertifiedTransaction, SignedTransaction, Transaction, VerifiedCertificate,
-        VerifiedSignedTransaction, VerifiedTransaction,
+
+    use crate::{
+        authority::{test_authority_builder::TestAuthorityBuilder, AuthorityState},
+        authority_aggregator::{AuthorityAggregator, AuthorityAggregatorBuilder},
+        authority_client::AuthorityAPI,
+        validator_tx_finalizer::ValidatorTxFinalizer,
```

Comment:
did your IDE do all this automatically? would be nice to revert unnecessary changes to preserve blame